app/views/mailers/reservation_mailer/car_reservation_updated.text.erb

```diff
@@ -23,3 +23,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url %>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```

Comment:
same review comment which is mentioned in _car_reservation_approved.html.erb_

Reply:
time fixed